.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: python
-python: 2.7
+python:
+  - 2.7
+  - 3.5
```

Comment:
confirmed, I think we probably want to pick a single python version to run tox, and control which python version is used for the tests using the tox_env environment variable